src/api/index.ts

```diff
@@ -47,3 +47,23 @@ export async function changeTaskColorRequest(
 ): Promise<void> {
   await axios.patch(urljoin(baseUrl, "tasks", taskId.toString()), { color });
 }
+
+export async function singUpRequest(
+  login: string,
+  password: string
+): Promise<void> {
+  await axios.post(urljoin(baseUrl, "users"), { login, password });
+}
+
+export interface User {
+  login: string;
+  password: string;
+  id: number;
+}
+
+export async function singInRequest(
+  login: string
+): Promise<AxiosResponse<User[]>> {
+  const userData = await axios.get(urljoin(baseUrl, `users?login=${login}`));
```

Comment:
why not search the exact user with login and password?
like ...?login=${login}&password=${password}

Reply:
I think it not nessesary. I'm wrong?